src/Portalum.Zvt/Models/StatusInformation.cs

```diff
@@ -6,6 +6,7 @@ namespace Portalum.Zvt.Models
     public class StatusInformation :
         IResponse,
         IResponseErrorMessage,
+        IResponseErrorCode,
```

Comment:
i have added the original blank error code, in order to clearly check for 
* payment success (== 0)
* or for example a forced EOD
previously we had to use the error string which is not as clean